src/main/java/org/grpc/service/entities/Shift.java

```diff
@@ -0,0 +1,61 @@
+package org.grpc.service.entities;
+
+import jakarta.persistence.*;
+import lombok.*;
+
+import java.io.Serializable;
+import java.time.LocalDateTime;
+
+@Entity
+@Getter
+@Setter
+@NoArgsConstructor
+@ToString
+@EqualsAndHashCode
+public class Shift implements Serializable {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long id;
+
+    private LocalDateTime startDateTime;
+    private LocalDateTime endDateTime;
+    private String typeOfShift;
+    private String shiftStatus;
+    private String description;
+    private String location;
+
+    @ManyToOne
+    @JoinColumn(name = "employee_id")
```

Comment:
What are the @ things in here and what they do?

Reply:
The @ things are called Annotations, and are there to tell other java classes how to cooperate with this code. The @s at the top are there to tell JPA that this is an Entity, and then to tell Lombok that it should generate code for getters, setters and so on. Later, they are options for the fields in the database, telling JPA which one should be the ID, and if there are any relations between the objects.